tests/README.md

```diff
@@ -0,0 +1,101 @@
+# E2E Tests
+
+This directory contains E2E tests which are run against BotKube installed on Kubernetes cluster and Slack API.
+
+## Prerequisites
+
+- Kubernetes cluster (e.g. local one created with `k3d`)
+- BotKube bot app configured for a given Slack workspace according to the [instruction](https://www.botkube.io/installation/slack/)
+- BotKube tester app configured according to the [instruction](#configure-tester-slack-application)
+
+### Configure Tester Slack application
+
+1. Create new Slack Application on [this page](https://api.slack.com/apps)
+2. Use "From an app manifest" option
+3. Copy and paste this manifest:
+
+    ```yaml
+    display_information:
+      name: BotKube tester
+    features:
+      bot_user:
+        display_name: Tester
+        always_online: false
+    oauth_config:
+      scopes:
+        bot:
+          - channels:join
+          - channels:manage
+          - chat:write
+          - users:read
+          - channels:history
+    settings:
+      org_deploy_enabled: false
+      socket_mode_enabled: false
+      token_rotation_enabled: false
+    ```
+
+4. Install this app into your workspace
+5. Navigate to the **OAuth & Permissions** section
+6. Copy the **Bot User OAuth Token** and save it for later.
```

Comment:
maybe it will be better to already export it as `SLACK_TESTER_APP_TOKEN`, so there will be no confiusion

Reply:
I can mention it here as well 👍